sr_system_info/src/sr_system_info/system_info.py

```diff
@@ -15,18 +15,17 @@
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import
+from builtins import round
```

Comment:
Do we need to import round when python already as a round that you can use without any import?

Reply:
Im not sure, I would of though pylint would of spotted something like this though

Reply:
It can't since we are overriding the default round function. If there's not specific reason about why using this specific round function I would rather use the one that natively comes with python

Reply:
The issue is I don't know why we used this specific round function it could have a reasoning behind it

Reply:
https://github.com/shadow-robot/sr_utils/pull/11

Reply:
I think it was required by the previous migration linter.... but we don't need that here!

Reply:
Removed it! :)